templates/workloads/services/lb-web/cf.yml

```diff
@@ -268,6 +268,31 @@ Resources:
       ServiceToken: !GetAtt RulePriorityFunction.Arn
       ListenerArn: !GetAtt EnvControllerAction.HTTPSListenerArn
 
+  HTTPListenerRuleWithDomain:
+    Type: AWS::ElasticLoadBalancingV2::ListenerRule
+    Condition: HTTPSLoadBalancer
+    Properties:
+      Actions:
+        - Type: redirect
+          RedirectConfig:
+            Protocol: HTTPS
+            Port: 443
+            Host: "#{host}"
+            Path: "/#{path}"
+            Query: "#{query}"
+            StatusCode: HTTP_301
+      Conditions:
+        - Field: 'host-header'
+          HostHeaderConfig:
+            Values:
+              - Fn::Join:
+                - '.'
+                - - !Ref WorkloadName
+                  - Fn::ImportValue:
+                      !Sub "${AppName}-${EnvName}-SubDomain"
```

Comment:
What is this saying exactly? 🤔   
if you go to `http://workload.env.app.domain.com/path` then redirect to https?

I thought what I wanted was something like if I go to `http://<anything>/path` redirect to `https://workload.env.app.domain.com/path` so that it also captures the default elb dns name:
```
      Actions:
        - Type: redirect
          RedirectConfig:
            Protocol: HTTPS
            Port: 443
            Host: 
             - Fn::Join:
                - '.'
                - - !Ref WorkloadName
                  - Fn::ImportValue:
                      !Sub "${AppName}-${EnvName}-SubDomain"
            Path: "/#{path}"
            Query: "#{query}"
            StatusCode: HTTP_301
      Conditions:
       - Field: 'path-pattern'
          PathPatternConfig:
            Values:
              !If
                - HTTPRootPath
                -
                  - "/*"
                -
                  - !Sub "/${RulePath}"
                  - !Sub "/${RulePath}/*"
```

Reply:
I think we might just want to redirect `http://frontend.env.app.domain.com/path` to `https://frontend.env.app.domain.com/path`, and `http://backend.env.app.domain.com/path` to `https://backend.env.app.domain.com/path`.